js/content/store.js

```diff
@@ -2051,9 +2048,9 @@ class AppPageClass extends StorePageClass {
         let nodes = document.querySelectorAll(".purchase_area_spacer");
         HTML.beforeEnd(nodes[nodes.length-1],
             `<div id="es_customize_btn" class="home_actions_ctn">
-                <div class="home_btn home_customize_btn" style="z-index: 13;">${ Localization.str.customize }</div>
-                <div class='home_viewsettings_popup'>
-                    <div class='home_viewsettings_instructions' style='font-size: 12px;'>${ Localization.str.apppage_sections }</div>
+                <div class="home_btn home_customize_btn app_tag" style="z-index: 13; margin-top: 3px;">${Localization.str.customize}</div>
```

Comment:
what's with the app_tag and margin-top?

Reply:
was added to make it look a bit nicer, but can be removed if you want.

Reply:
Yes please, it adds unneccesary margins, makes even bigger space around it than there already is, I think